docs/beamrunner/reference.md

```diff
@@ -95,9 +115,9 @@ For the full list of pipeline options, enter  `--help=StreamsPipelineOptions` on
 
 | Parameter | Description | Default value |
 | --- | --- | --- |
-| `restUrl` | The URL for the REST API when you use the `DISTRIBUTED` context type. This parameter is required if you implement metrics in the application. | [null] |
-| `userName` | The user name for basic authentication with REST API when you use the `DISTRIBUTED` context type. | [null] |
-| `userPassword` | The user password for basic authentication for REST API when you use the `DISTRIBUTED` context type. | [null] |
+| `restUrl` | The URL for the REST API when you use the `DISTRIBUTED` context type. This parameter is required if you implement metrics in the application. | The return value of command `streamtool geturl` |
+| `userName` | The user name for basic authentication with REST API when you use the `DISTRIBUTED` context type. | The `user.name` system property |
+| `userPassword` | The user password for basic authentication for REST API when you use the `DISTRIBUTED` context type. If the option value is a path of a readable file (recommended) the password used will be read from first line of the file. Otherwise the option value is used as the password. | [null] |

```

Comment:
"A path to a file containing the user password (recommended) or a string containing the user password for basic authentication... . If the option value is a path to a readable file, the first line of the file will be used as the password."